src/parsers/gas.ts

```diff
@@ -133,6 +118,31 @@ export const parseRainbowMeteorologyData = (
     confirmationTimeByPriorityFee,
     currentBaseFee,
   } = rainbowMeterologyData.data;
+
+  // API compatible
+  let blocksToConfirmation: BlocksToConfirmation;
+  if (!rainbowMeterologyData.data.blocksToConfirmation) {
+    const byPriorityFee: BlocksToConfirmationByPriorityFee = {
+      1: confirmationTimeByPriorityFee[15],
+      2: confirmationTimeByPriorityFee[30],
+      3: confirmationTimeByPriorityFee[45],
+      4: confirmationTimeByPriorityFee[60],
```

Comment:
i changed this from seconds to number of blocks because is cleaner and we needed to add these values for the max base fee changing the api, the problem with this is that we'll have to maintain this for backwards compatibility

Reply:
hey once the change is live on the fees estimation API, we can remove this part of the code too correct? (only backend needs to worry about backwards compatibility, not the client)

Reply:
yeah